### ISSUE TITLE
i am trying to develop android app but the app keeps on crashing 

### ISSUE THREAD

**Issue by knikhil52** [Azure-Samples/cognitive-services-android-customvision-sample#8]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
>

### Any log messages given by the failure
>

### Expected/desired behavior
>app crashes instantly when its opened

### OS and Version? 
> android 6.0.1 api 23

### Versions
>

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by djkormo** [comment]
The same problem on my HTC  Desire 630 Dual Android 6.0.1.

**Comment by muhtadi** [comment]
same problem djkormo

**Comment by djkormo** [comment]
Waiting for fix .....

**Comment by bradlindblad** [comment]
Same, crashes instantly for me, too.

**Comment by djkormo** [comment]
Here you have working similar repo.
https://github.com/MindorksOpenSource/AndroidTensorFlowMachineLearningExample

**Comment by DengYiping** [comment]
Same Issue here. Both crashed on my emulator and HTC phone, also on Google Glasses. A bit of debugging is needed.

**Comment by DengYiping** [comment]
I did a bit of research on the following. The problem is caused by `ImageUtils.java file`.
The file have some Native method call, and somehow it doesn't match the signature. Maybe consider recompile the `c` files. Cheers.

**Comment by DengYiping** [comment]
Also, reference to the C file here: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/android/jni/imageutils_jni.cc

**Comment by ksujanpaul** [comment]
the application is crashing immediately when we open it

**Comment by DengYiping** [comment]
@ksujanpaul  I think part of this project's code is forked from the example of tensorflow, and for tensorflow there are some JNI code included. Those codes are not correctly compiled here, and I think that is the problem here.

**Comment by raulfmiranda** [comment]
Here, it is also crashing immediately when it is opened. It crashes at line 110 of CameraActivity.java:
`ImageUtils.convertYUV420SPToARGB8888(bytes, rgbBytes, previewWidth, previewHeight, false);`

```
W/tensorflow: ImageUtils: Native library not found, native RGB -> YUV conversion may be unavailable.
E/art: No implementation found for void demo.tensorflow.org.customvision_sample.env.ImageUtils.convertYUV420SPToARGB8888(byte[], int[], int, int, boolean) 
```

@DengYiping how can I recompile the c files? Can you explain how to do that?

**Comment by DengYiping** [comment]
@raulfmiranda  Please refer to the original repo from TensorFlow. This project basically just fork the project from Google and build the app on top.

**Comment by dlfu** [comment]
closed by PR #17
